frontend/lib/identification/base_card_details.dart

```diff
@@ -2,15 +2,15 @@ enum CardType { standard, gold }
 
 class BaseCardDetails {
   final String fullName;
-  final String hashSecretBase64;
+  final List<int> pepper;
```

Comment:
Shouldn't this be a UInt8Array or so?

Reply:
Nope, it seems like binary data ist List<int> in dart

Reply:
There is Uint8List though which seems more appropriate at first glance.
https://api.dart.dev/be/180791/dart-typed_data/Uint8List-class.html

Reply:
Ah, weirdly, Uint8List implements List<int>. Wild.